### PR TITLE
[WIP] fetching metadata added to bundle downloader

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [ ] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] Automatic tests added/update
- [ ] Automatic tests pass
- [ ] Software runs (locally or hosted)
- [ ] Sanity check passed
- [ ] Self-review passed

### CODE REVIEW

File: src/services/data_model_engine.js [hunk 1]
```diff
@@ -226,13 +226,28 @@ export default class DataModelEngine {
...
+      throw new Error('Could not fetch the bundle metadata from the shelterer');
+    }
+
     const bundle = await this.bundleDownloader.downloadBundle(nodeUrl, bundleId);
     if (!bundle) {
       throw new Error('Could not fetch the bundle from the shelterer');
     }
+
     this.bundleBuilder.validateBundle(bundle);
     await this.uploadRepository.verifyBundle(bundle);
```
Comment (spherefoundry) [inline src/services/data_model_engine.js]: IMO: We should also validate the bundle metadata. Could you add a task for this in Asana?
